sequencer/src/options.rs

```diff
@@ -87,6 +102,19 @@ impl Options {
     }
 }
 
+#[derive(Clone, Debug, Snafu)]
+struct ParseDurationError {
+    reason: String,
+}
+
+fn parse_duration(s: &str) -> Result<Duration, ParseDurationError> {
```

Comment:
cc @jbearer - is there a better place I can put this?

Reply:
I think this is a fine place for it. You could make it `pub` and import it from the orchestrator, so it's not duplicated. But not a big deal